test/test_api/test_api.py

```diff
@@ -272,17 +279,20 @@ def test_tabular_regression(openml_name, resampling_strategy, backend):
 
     # Search for an existing run key in disc. A individual model might have
     # a timeout and hence was not written to disc
+    successful_num_run = None
+    SUCCESS = False
     for i, (run_key, value) in enumerate(estimator.run_history.data.items()):
-        if 'SUCCESS' not in str(value.status):
```

Comment:
Could you unite this process?  We have the two identical processings. (Later should be fine.)

Reply:
do you mean the process in test_tabular_classification and test_tabular_regression? I'm sorry I did not understand which two processes to unite

Reply:
Yes, yes those tests. It has the same process, right? (but since it is a test code, if you think it is not necessary, it is also ok to ignore this comment)

Reply:
yeah there are subtle differences in 1 or 2 lines so I think its fine to not unite them